dev-docs/howtoguides/testing.md

```diff
@@ -60,19 +60,27 @@ The testing can be overridden to install ubuntu-advantage-tools from other sourc
 To run the tests, you can use `tox`:
 
 ```shell
-tox -e behave-lxd-20.04
+tox -e behave
 ```
 
 or, if you just want to run a specific file, or a test within a file:
 
 ```shell
-tox -e behave-lxd-20.04 features/unattached_commands.feature
-tox -e behave-lxd-20.04 features/unattached_commands.feature:28
+tox -e behave features/unattached_commands.feature
+tox -e behave features/unattached_commands.feature:28
 ```
 
-As can be seen, this will run behave tests only for release 20.04 (Focal Fossa). We are currently
-supporting 6 distinct releases:
+or, if you want to run a specific file for a specific release and machine type, or a specific test:
 
+```shell
+tox -e behave -- features/config.feature -D releases=jammy -D machine_types=lxd-container
+tox -e behave -- features/config.feature:132 -D releases=jammy -D machine_types=lxd-vm
+```
+
+As can be seen, this will run behave tests only for release 22.04 (Jammy Jellyfish). We are currently
+supporting 7 distinct releases:
+
+* 23.10 (Mantic Minotaur)
```

Comment:
We no longer run tests for Lunar and Kinetic, we can remove those entries

Reply:
Please avoid specifying the releases, unless if giving an example. Specify the rule:
- all supported releases
- LTS releases, including those under ESM
- devel release, at some point in the development cycle

Otherwise this is going to go stale often and we need to update it every 3 months or so during the interim cycles

Reply:
Updated to mention the rule instead of specific releases